.circleci/config.yml

```diff
@@ -478,6 +512,21 @@ jobs:
                 working_directory: ~/workbench/api
                 command: ./project.rb bigquerytest
 
+  api-contract-test:
```

Comment:
This runs the contract tests and then persists the ~/workbench/api/build/pacts directory so that the generated contracts can be uploaded in  `api-contract-upload`